src/app/create-email-template/[sessionId]/page.tsx

```diff
@@ -1,7 +1,18 @@
-'use server';
-
 import { CreateEmailTemplate } from '@forms/CreateEmailTemplate/CreateEmailTemplate';
+import { PageProps, TemplateType } from '@utils/types';
+import { getSession } from '@utils/form-actions';
+import { redirect, RedirectType } from 'next/navigation';
+
+const CreateEmailTemplatePage = async ({
+  params: { sessionId },
+}: PageProps) => {
+  const session = await getSession(sessionId);
+
+  if (!session || session.templateType !== TemplateType.EMAIL) {
```

Comment:
If we are going to have this type check then can you ensure that we have it consistently for all channels (if not already).

Reply:
I believe @bhansell1 has already done this for SMS and NHS app too. If not I'll capture in tidy up ticket